setup.py

```diff
@@ -61,8 +61,8 @@
     include_package_data=True,
     url='https://github.com/certtools/intelmq/',
     license='AGPLv3',
-    description='IntelMQ is a solution for CERTs to process data feeds, '
-                'pastebins, tweets throught a message queue.',
+    description='IntelMQ is a solution for IT security teams for collecting and '
+                'processing security feeds throught a message queue.',
```

Comment:
"throughout" does not sound like proper english  for my ears. Sorry, no. 
"using a" or "with message queues". Anyway, these are really very minor changes.